src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -438,6 +446,7 @@ CREATE TABLE courthouse
 (cth_id                      INTEGER                       NOT NULL
 ,courthouse_code             INTEGER                       NOT NULL          UNIQUE

```

Comment:
Is INTEGER the correct type for this column - `courthouse_code`?

Reply:
what data type is expected ?
moj_courthouse_s.c_code is varchar in legacy and contains the full text courthouse_name, hence is being mapped to courthouse_name ( c_id in legacy is an incomplete list of the same full courthouse names).
The numeric courthouse code is found in moj_daily_list_s.c_crown_court_code and this is the intended content of courthouse_code, as per the column level comment defined on this column in the schema creation script. In legacy it takes integer values between 401 and 762.